crates/wasmprinter/src/lib.rs

```diff
@@ -2882,7 +2882,7 @@ impl Printer {
                     self.print_idx(&state.module_names, index)?;
                     self.end_group();
                     for arg in args.iter() {
-                        self.result.push(' ');
+                        self.newline();
```

Comment:
I wonder if we should also do a `self.newline()`above that separates the `instance` group with the `instantiate` group (both here and for `instantiate component` below).

That would turn:

```wat
(instance (;7;) (instantiate (module 0)
    (with "bar" (instance 4))
    (with "baz" (instance 5))
    (with "foo" (instance 6))
  )
)
```

into

```wat
(instance (;7;)
  (instantiate (module 0)
    (with "bar" (instance 4))
    (with "baz" (instance 5))
    (with "foo" (instance 6))
  )
)
```

What are your thoughts?

Reply:
Although that would mean an instantiation with arguments spans multiple lines.

Let's leave it as you have now, can always address later.